CITATION.cff

```diff
@@ -0,0 +1,32 @@
+cff-version: 1.2.0
+title: NOMAD Simulations Schema Plugin
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Nathan
+    family-names: Daelman
+    orcid: 'https://orcid.org/0000-0002-7647-1816'
+  - given-names: Bernadette
+    family-names: Mohr
+    orcid: 'https://orcid.org/0000-0003-0903-0073'
+  - given-names: José M.
+    family-names: Pizarro
+    orcid: 'https://orcid.org/0000-0002-6751-8192'
+  - given-names: Esma B.
+    family-names: Boydas
+    orcid: ''
+  - given-names: Joseph F.
+    family-names: Rudzinski
+    orcid: 'https://orcid.org/0000-0003-3403-640X'
+doi:
```

Comment:
i guess here you need to add the zenodo doi?

Reply:
yeah, but it seems a bit cyclic, I need to make a release to get the DOI, and then add it here after which will then be present in the next release :shrug:

Reply:
Ok, no problem. Then good as it is, then later we change doi and README with the badge

Reply:
I think I found a way to add the badge before the first release, I followed this: https://gist.github.com/seignovert/ae6771f400ca464d294261f42900823a

Reply:
Ok cool. Would you mind adding at the begining with the other badges? 

We could also add a citation before the contacts table for `## How to cite this work`